### PR TITLE
Update automatic-registration.md

### PR DESCRIPTION
**Pull request by tktracker**

### CODE REVIEW

**Comment by PRMerger18** [comment]
@tktracker : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by aczechowski** [comment]
Started email with Lee & Juanita, since there's a history here via #2175 from MVP

**Comment by frankroj** [comment]
Rejecting this request. There are several other PRs where this verbiage was asked to be added in (see [PR2175](https://github.com/MicrosoftDocs/memdocs/issues/2175), [PR2181](https://github.com/MicrosoftDocs/memdocs/issues/2181), [PR3200](https://github.com/MicrosoftDocs/memdocs/issues/3200), and [PR6072](https://github.com/MicrosoftDocs/memdocs-pr/pull/6072). If the PM (@JuanitaBaptiste ) wants to make changes to this doc, please request that she submit a PR.

Also @tktracker, when editing docs, keep comments out of the docs of what you want done. Instead, just make the edit that you want to see. You can comment in the PR or directly in the article via the Feedback section in the article.

**Comment by frankroj** [comment]
Before officially rejecting and closing PR and issue, reached out to @tktracker to see if there is a way that instead of removing the section, we can reword it to avoid confusion.

**Comment by traya1** [comment]
Learn Build status updates of commit _[7cf568a](https://github.com/tktracker/memdocs/commits/7cf568a765f930bb9766cfd2c06623d0f592d8c1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/autopilot/automatic-registration.md](https://github.com/tktracker/memdocs/blob/patch-2/memdocs/autopilot/automatic-registration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212272000510707-2443/BuildReport?accessString=fb661b32544c69034babcf3cf625f50e9be194fc24e6ce41f666a2941c63a1de).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by traya1** [comment]
Learn Build status updates of commit _[ea06c45](https://github.com/tktracker/memdocs/commits/ea06c4541269050c0708a8e3a0d63cf2c2a9a14a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/autopilot/automatic-registration.md](https://github.com/tktracker/memdocs/blob/patch-2/memdocs/autopilot/automatic-registration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212272009043793-2443/BuildReport?accessString=44533a55d993b86570b6226768c40936fdc487748fea17f5790803bb2762f64c).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by traya1** [comment]
Learn Build status updates of commit _[0e34488](https://github.com/tktracker/memdocs/commits/0e34488187e8f359d7ea6aa281a788a9b0e1a137)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/autopilot/automatic-registration.md](https://github.com/tktracker/memdocs/blob/patch-2/memdocs/autopilot/automatic-registration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212272024385150-2443/BuildReport?accessString=aa98a19b379957ad5aa785c10748e65db7823981603f9ec3261d03a3008ef7bf).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by frankroj** [comment]
Waiting on feedback from AutoPilot PM before publishing.

**Comment by traya1** [comment]
Learn Build status updates of commit _[415c88a](https://github.com/tktracker/memdocs/commits/415c88aec9e1afe6e964fc6220dd1a5748911e59)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/autopilot/automatic-registration.md](https://github.com/tktracker/memdocs/blob/patch-2/memdocs/autopilot/automatic-registration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212281409523908-2443/BuildReport?accessString=de183085d203900fdd9a6d010a992db6ed04568443dba93a6da189905dce5f05).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by traya1** [comment]
Learn Build status updates of commit _[8464ef7](https://github.com/tktracker/memdocs/commits/8464ef7c6a5e399d3b800a66a917d9ad236e044e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/autopilot/automatic-registration.md](https://github.com/tktracker/memdocs/blob/patch-2/memdocs/autopilot/automatic-registration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212281412418001-2443/BuildReport?accessString=86e7e53e6db26faf62658aeac27489ee7f85058b13f392ef05ddc713b81bdf21).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by frankroj** [comment]
PM approved change. Signing off and approving request and merging.

**Comment by frankroj** [comment]
#sign-off
